bathy_dave_plugin/world/bathy_dave.world

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0"?>
+<sdf version="1.5">
+  <world name="default">
+
+    <gui>
+      <camera name="user_camera">
+        <view_controller>ortho</view_controller>
+        <projection_type>orthographic</projection_type>
+        <clip>0.1 5000</clip>
+        <pose>485397.09375 58126.83 8.788642 0 0.523999 1.590999</pose>
+      </camera>
```

Comment:
I also moved the camera so the ROV is visible when the simulation launches. I'm not sure whether this will work for you: I'm finding that the "clip" settings have no effect and I'm unable to set the initial zoom of the camera. This could be because my version of Gazebo is not the very latest, so if you find that this position doesn't show the ROV when you run it, please change it back.

Reply:
Works perfect.